cmd/titond/main.go

```diff
@@ -77,3 +93,46 @@ func titond(ctx *cli.Context) error {
 
 	return nil
 }
+
+func checkSwagger(ctx *cli.Context) error {
```

Comment:
I think this way is so good. But how about making test directory? Then, we can seperate to this logic with our app.

Reply:
I think it is good idea. let me move the logic to test directory

Reply:
I updated!